apm-sniffer/apm-agent-core/src/test/java/org/apache/skywalking/apm/agent/core/plugin/witness/WitnessTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.apm.agent.core.plugin.witness;
+
+import net.bytebuddy.description.method.MethodDescription;
+import net.bytebuddy.matcher.ElementMatcher;
+import net.bytebuddy.matcher.ElementMatchers;
+import org.apache.skywalking.apm.agent.core.plugin.WitnessFinder;
+import org.apache.skywalking.apm.agent.core.plugin.WitnessMethod;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * unit test for WitnessFinder
+ */
+public class WitnessTest {
+
+    private String className = "org.apache.skywalking.apm.agent.core.plugin.witness.WitnessTest";
```

Comment:
Using `WitnessTest#getClass` should be better?

Reply:
I am not sure If I can use WitnessTest.class.
I think the witness class should not be loaded before the app start.

Reply:
I am not asking you to change the mechanism. My point is this test class. This field doesn't need to be a hard coded string. You could use `getClass.getName` to instead of, right?

Reply:
Yes,you are right.
But it's good for the unit test, not good as demo for users.We should find a banlance.

Reply:
Good point, let's keep this unchanged.